_rules/image-accessible-name-descriptive-qt1vmo.md

```diff
@@ -253,7 +253,7 @@ This `img` element has no [accessible name][] because it is not [included in the
 This `svg` element is ignored because it is a child of a link that provides its [accessible name][].
 
 ```html
-<a href="//w3.org" aria-label="W3C Website">
+<a href="https://w3.org" aria-label="W3C Website">
```

Comment:
Why does that need to change? Protocols aren't required in URLs. Let the browser figure it out, they're good at that.

Reply:
They really aren't though: https://www.paulirish.com/2010/the-protocol-relative-url/

Reply:
Yep, every recent reference I could find (including that one) considered that protocol relative URLs are an anti-pattern since SSL and HTTPS have wide support. Which is why I decided to ditch them. (for example, Wikipedia decided to ditch them back in 2015)